Tests/DependencyInjection/Compiler/TwigFormPassTest.php

```diff
@@ -5,6 +5,9 @@
 use Norzechowicz\AceEditorBundle\DependencyInjection\Compiler\TwigFormPass;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
+/**
+ * @author azzra <azzra@users.noreply.github.com>
```

Comment:
Agree, revert those authors that you added in current PR and I will create new PR that will remove them from all other files